peekaboo/daemon.py

```diff
@@ -377,6 +375,10 @@ def run():
     sig_handler = SignalHandler()
     sig_handler.register_listener(cuckoo)
 
+    if not cuckoo.start_tracker():
```

Comment:
Yes, up until now we had an implicit dependency on Cuckoo. With embed mode it was obvious, since we started Cuckoo as a child process. With API mode it was more subtle since we checked connectivity at startup and refused to start if it was unavailable. The above change maintains that dependency.

But I am jut now working on a change that only starts the Cuckoo job tracker if any rule uses Cuckoo. This is statically configured for the Cuckoo rules and for the expression rule it determines whether any expression uses the `cuckooreport` identifier. With that change, Cuckoo and its REST API being accessible will be come fully optional. (But *if* a rule uses it, it must be available just as before.)